warehouse/routes.py

```diff
@@ -178,7 +178,7 @@ def includeme(config):
         domain=warehouse,
     )
     config.add_pypi_action_redirect(
-        "packages_rss",
-        "/rss/packages.xml",
+        "projects_rss",
```

Comment:
This is an existing API that's redirecting  to a new URL, we can't change this name or else all the old links will be broken.